foursight_core/react/api/aws_s3.py

```diff
@@ -0,0 +1,45 @@
+import boto3
+from .datetime_utils import convert_utc_datetime_to_useastern_datetime_string
+
+
+class AwsS3:
+
+    @staticmethod
+    def get_buckets() -> list:
+        results = []
+        try:
+            s3 = boto3.resource("s3")
+            results = sorted([bucket.name for bucket in s3.buckets.all()])
+        except Exception as e:
+            print("Get buckets error: " + str(e))
```

Comment:
It's worth using f-strings where possible both because they're more concise and because we've seen it claimed they're way more efficient than string addition for some reason.
```suggestion
            print(f"Get buckets error: {e}")
```
That said, I suspect this should not be a call to `print` at all. If it were, I'd be telling you to rewrite it to `PRINT` from `dcicutils.misc_utils`. But I think logging this kind of message is best. If you think a traceback is helpful, make sure to use `.exception()` to do the logging, or otherwise presumably `.error()` for whatever logger you use.

Reply:
+1 for f-strings

Reply:
All set.